projects/storefrontapp-e2e-cypress/cypress/helpers/quote.ts

```diff
@@ -978,11 +993,9 @@ function checkCommentAmountChanged() {
  * Gets the current amount of displayed comments.
  */
 function getCommentAmount() {
```

Comment:
relates to all the comment-related methods: instead of suffix Amount i would use suffix NumberOf so. Amount sound strange here
* getCommentAmount -> getNumberOfComments
* checkCommentAmountChanged -> checkNumberOfCommentsChanged
* ....

Reply:
changed several occurrences of "amount" to "number".